rest-api/src/main/resources/definitions/account_summary_search.yml

```diff
@@ -46,6 +46,9 @@ properties:
     adminOnly:
         description: Administrative accounts are accounts with any roles that allow them to operate against our administrative APIs (not participant-facing, and not requiring consent). When null, the search returns all accounts. When true, returns accounts with at least one assigned role. When false, returns accounts with no administrative roles.
         type: boolean
+    enrolledInStudyId:
+        description: If no study ID is supplied, all accounts will be returnd. If an ID is supplied, only accounts assigned to that study are returned.
```

Comment:
typo: "returned"

Reply:
Note that this is also retrievable via /v5/studies/{studyId}/participants/search, which also takes an AccountSummarySearch. Only researchers and study coordinators of the appropriate study can call that method.

Reply:
Thanks, Alx!